src/ip/Intercept.cc

```diff
@@ -204,7 +204,7 @@ Ip::Intercept::IpfInterception(const Comm::ConnectionPointer &newConn, int silen
     memset(&natLookup, 0, sizeof(natLookup));
     // for NAT lookup set local and remote IP:port's
     if (newConn->remote.isIPv6()) {
-#if IPFILTER_VERSION < 5000003
+#if HAVE_NATLOOKUP_NL_INIPADDR_IN6
         // warn once every 10 at critical level, then push down a level each repeated event
         static int warningLevel = DBG_CRITICAL;
         debugs(89, warningLevel, "IPF (IPFilter v4) NAT does not support IPv6. Please upgrade to IPFilter v5.1");
```

Comment:
Since we are no longer relying on the IPFilter version for this check, should we replace "IPF (IPFilter v4)" with something like "Your IPF" or "Your IPFilter"?